kubeflow/common/iap-ingress/base/config-map.yaml

```diff
@@ -167,11 +167,12 @@ data:
       echo health check URI is ${HEALTH_CHECK_URI}
 
       # See: https://cloud.google.com/service-mesh/docs/iap-integration#deploying_the_load_balancer
-      HC_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="status-port")].nodePort}')
+      HC_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
       echo Setting BackendConfig healthCheck.port to: ${HC_INGRESS_PORT}
       kubectl patch backendconfig iap-backendconfig -n ${NAMESPACE} --type json -p '[{"op": "replace", "path": "/spec/healthCheck/port", "value": '${HC_INGRESS_PORT}'}]'
       
-      HC_INGRESS_PATH=$(kubectl -n istio-system get deployments istio-ingressgateway -o jsonpath='{.spec.template.spec.containers[?(@.name=="istio-proxy")].readinessProbe.httpGet.path}')
```

Comment:
We shouldn't hardcode it, can you parse the Deployment yaml for istio-ingressgateway here?

Reply:
Reverted